lpc845/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name    = "lpc845-pac"
```

Comment:
Does it make sense to have a PAC just for the LPC845? I haven't checked the LPC84x files, but for LPC82x, there are no differences between the SVD files, beside the name.

Reply:
Just checked: The 844 seems to be a subset, missing:
- USART2, USART3, USART4
- capacitive touch
- I2C2, I2C3
- DACS
but otherwise identical. 
I think it's better to keep it separate, but I'm not that sure about it.

Reply:
Okay, since there are significant differences, I agree that it's better to keep them separate. Thanks for checking!